extension/generate-config.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Please excuse if I might be somewhat allergic to the term "config". 
Could this script and its name be another hint that there is something like a "plugin" / "integration" / "adapter" or "description" required in order to enable the "cloud-storage-extension" to be managed with the extension manager? 
If the answer is yes, then maybe it's worth to coin a phrase for this piece of software/config etc.

We also should describe the life cycle of this config: 
- where does it come from, when / triggered by what is it generated, by whom?
- what is the syntax or container format? (apparently typescript) 
- what is the content? (apparently file name, size and version of the jar)
- who is the receiver and what are the use cases?

Reply:
renamed to description